renovate.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Maybe you didn't see my comment on the other PR, but what is Renovate going to update on this repo ? Some GitHub actions?

Reply:
> what is Renovate going to update on this repo ? Some GitHub actions?

Yes, if we're going to add an external GitHub action, this would be updated via renovate.
However, at the moment we don't use any external GitHub action, therefore nothing is being updated.

I added info to the commit message.